src/common/interceptor/request/offsetPagination.interceptor.ts

```
@@ -93,6 +101,16 @@ export function OffsetPaginate(
         }),
       );
     }
+
+    private attachBoxIdToRequest(request: Request) {
+      const token = request.headers.authorization?.split(' ')[1];
+      try {
+        const payload: BoxUser = this.jwtService.verify(token);
+        request['user'] = new BoxUser(payload);
+      } catch (e) {
+        void e;
```

I do not expect that so often will execute this line, but in general not a good approach to hide an exception in this way. Why you don't log it or rethrow?